repose-aggregator/core/repose-core/src/main/scala/org/openrepose/powerfilter/ReposeFilterChain.scala

```diff
@@ -111,8 +112,12 @@ class ReposeFilterChain(val filterChain: List[FilterContext], originalChain: Fil
   def doMetrics(filter: String)(requestProcess: (HttpServletRequest, HttpServletResponse) => Unit): (HttpServletRequest, HttpServletResponse) => Unit = (request, response) => {
     val startTime = System.currentTimeMillis()
 
+    val scope = tracer.buildSpan(s"Filter $filter").startActive(true)
```

Comment:
The version [Jager version we're currently using](https://github.com/jaegertracing/jaeger-client-java/blob/d9b81ea71d43097d02dfde296c6f1602645a40cd/jaeger-core/src/main/java/io/jaegertracing/Span.java#L144) uses a `-`, but that is a moot point since we are moving from that version at some point.